en/docs/deploy-and-publish/deploy-on-gateway/api-gateway/gateway-policies.md

```diff
@@ -0,0 +1,98 @@
+
+!!! warning
+    A specific gateway with active global policy deployments will apply to all the APIs deployed on that gateway. Similarly, this applies to tenants within the tenant available setup. Global policies will be applied to all APIs belonging to a specific gateway in the tenant corresponding to the logged-in tenant during the deployment of the global policy.
+
+!!!info
+    Although all managing actions can be taken by Admin users with specific scopes, API creators and publishers can view the meta information of global policies such as names, descriptions, policy mappings, and deployed gateway labels.
+
+## Step 1 - Create a gateway policy
+
+Follow the instructions provided below to create a gateway policy:
+
+1. {!includes/sign-in-publisher.md!}
+
+2. Navigate to **Global Policies** in the left menu and proceed to select **Global Policies**.
+
+    The **Create A New Global Policy** page appears.
+
+    [![Global policies startup]({{base_path}}/assets/img/deploy/gateway/global-polices-fresh-start.png){: style="width:90%"}]({{base_path}}/assets/img/deploy/gateway/global-polices-fresh-start.png)
+
+3. Update the policy name and description, then simply drag and drop the desired policies onto the respective flows to attach them.
+
+    Let's create a sample gateway policy mapping following the steps below:
+
+    - Navigate to the Request tab of the Policy List.
+    - Drag the **Add Header** policy and drop it into the Request Flow dropzone. This action triggers a side panel to appear on the right-hand side.
+    - Fill in the necessary details using the provided values below.
+
+        <table>
+            <tr>
+                <th>Field</th>
+                <th>Sample Value</th>
+            </tr>
+            <tr>
+                <td>Header Name</td>
+                <td>Foo</td>
+            </tr>
+            <tr>
+                <td>Header Value</td>
+                <td>Bar</td>
+            </tr>
+        </table>
+
+    - Finally, click **Save** to complete the process.
+
+        [![Save individual policy data]({{base_path}}/assets/img/deploy/gateway/global-polices-add-header-policy.png){: style="width:90%"}]({{base_path}}/assets/img/deploy/gateway/global-polices-add-header-policy.png)
+
+        !!!note
+            All the policies listed for addition as global policies are [common policies]({{base_path}}/design/api-policies/create-policy/).
+
+    - After saving the dropped policy, you should now be able to view the associated **Add Header** policy indicated by the initials **AH**.
+
+    - If you click on the recently attached **AH (Add Header)** policy, you'll still have access to view and edit the initially entered values.
+
+        [![View added common policy]({{base_path}}/assets/img/deploy/gateway/global-polices-add-header-policy-view.png){: style="width:90%"}]({{base_path}}/assets/img/deploy/gateway/global-polices-add-header-policy-view.png)
```

Comment:
This screenshot can be removed.

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/7526/commits/64bafa0aa418ed0dbbc05491694766f9d24f000b.
Thank you for your suggestion.